en/full-node.md

```diff
@@ -190,7 +190,7 @@ have an easy-to-use node.
 * Desktop or laptop hardware running recent versions of Windows, Mac OS
   X, or Linux.
 
-* {{site.text.bitcoin_datadir_gb}} gigabytes of free disk space,
+* {{site.text.bitcoin_datadir_gb_pruned}} gigabytes of free disk space,
   accessible at a minimum read/write speed of 100 MB/s.
```

Comment:
Maybe drop this minimum speed bit too. Most are fast enough, and there isn't strictly a minimum...

Reply:
I see. Let me know what speed to drop it too.

Reply:
Most people have no idea how to check their disk read/write speed, so I imagine they would ignore this requirement.